NeoML/include/NeoML/Dnn/Dnn.h

```diff
@@ -635,6 +641,9 @@ class NEOML_API CDnn : public CDnnLayerGraph {
 	bool autoRestartMode;
 	// The low memory use mode
 	bool isReuseMemoryMode;
+	
+	// Reference information 
+	CPtr<CDnnRefManager> refInfo;
```

Comment:
so var name should be `referenceDnnCounter` or `referenceDnnRegistoror` or `referenceDnnManager`